client/src/helpers/getIssuesBySearchText.helper.ts

```diff
@@ -0,0 +1,13 @@
+import memoizeOne from 'memoize-one';
+
+export type GetIssuesForSprint = {
+	(searchString: string, issues: WebApi.Entities.Issue[]): WebApi.Entities.Issue[];
+};
+
+const getIssuesForSprint: GetIssuesForSprint = (searchString, issues) => {
+	return !!issues?.length ? issues.filter((issue) => issue.summary?.toLowerCase().includes(searchString)) : [];
+};
```

Comment:
issues are always have type WebApi.Entities.Issue[].
As it is an array, we don't need to do additional checking !!issues?.length ? 
return issues.filter((issue) => issue.summary?.toLowerCase().includes(searchString));

Reply:
@danylokarpenko 

Changed in - https://github.com/BinaryStudioAcademy/bsa-2020-TaskyHusky/pull/233/commits/d463d05e870e13ca87a8513588b9db867426e3c3

`issues` can be undefined, since we first load sprints, render them, once we have sprints loaded, we load issues for sprints, and render issues after. 

If issues are `undefined` we return an empty array instead.

Reply:
ok, changed type.